graviton/blackbox.py

```diff
@@ -260,1287 +274,1269 @@ def final_as_row(self) -> dict:
         }
 
 
-class DryRunEncoder:
-    """Encoding utilities for dry run executions and results"""
+class DryRunInspector:
```

Comment:
moving class up which causes a big diff

Reply:
@tzaffi For future PRs - As a reviewer, I prefer to isolate source code moves, particularly when the PR otherwise makes extensive changes.  As presented, it complicates the review process because I feel I can't rely on the diff.